src/components/game/Sidebar/Player.js

```diff
@@ -33,14 +34,14 @@ const Player = ({
   const { displayName, photoURL } = profile;
 
   return (
-    <Segment disabled={!isActive && false}>
+    <Segment disabled={!isConnected && false}>
       <Feed>
         <Feed.Event>
           <Feed.Label image={photoURL} />
           <Feed.Content>
             <Feed.Date>
               {displayName} {isCurrentPlayer && <span>({t("sidebar.player.current")})</span>}{" "}
-              {!isActive && <span>({t("sidebar.player.disconnected")})</span>}
+              {!isConnected && <span>({t("sidebar.player.disconnected")})</span>}
```

Comment:
Maybe we could gray-out disconnected players?

Reply:
it marks them as disconnected as soon as they disconnect from the socket, so even if you refresh the page, you will be disconnected for a second, so graying-out may introduce some flickering